CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Changed
+- Changed version from 3.18.1-2 to 3.19.1-1
```

Comment:
Just describe the functional changes; in this case a Nexus upgrade to version 3.19.1. The dogu-specific version (with the "-1") will be added later.